modules/dispatcher/dispatcher.c

```diff
@@ -1365,7 +1377,7 @@ static struct mi_root* ds_mi_list(struct mi_root* cmd_tree, void* param)
 				9, part_it->name.s, part_it->name.len);
 
 		if (part_node == NULL
-			|| ds_print_mi_list(part_node, part_it) < 0) {
+			|| ds_print_mi_list(part_node, part_it, explicit) < 0) {
```

Comment:
as third param of ds_print_mi_list() is a bitmask, use here the predefined flags as macro (see the other comment). Maybe "explicit" is not the best name - better use a "flag" variable to collect through the code all the flags you want to pass to ds_print_mi_list().